packages/react-tc-auth/src/modules/storage/LocalStorageService.test.ts

```diff
@@ -5,8 +5,8 @@ const STORAGE_TOKEN_VALUE_ACCESSOR = 'value';
 const STORAGE_TOKEN_EXPIRES_ACCESSOR = 'expiresInUtc';
 
 const currentYear = new Date().getFullYear();
-const NEXT_YEAR_EXPIRATION_DATE = new Date(new Date().setYear(currentYear + 1)).toISOString();
-const LAST_YEAR_EXPIRATION_DATE = new Date(new Date().setYear(currentYear - 1)).toISOString();
+const NEXT_YEAR_EXPIRATION_DATE = new Date(new Date().setFullYear(currentYear + 1)).toISOString();
+const LAST_YEAR_EXPIRATION_DATE = new Date(new Date().setFullYear(currentYear - 1)).toISOString();
```

Comment:
And what is the semantic difference between setYear and setFullYear? It seems that it is impossible to set an incomplete year anyway